### ISSUE TITLE
Familiarize myself with MTL and the research it supports.

### ISSUE THREAD

**Issue by ljmoody** [ljmoody/myrepo#1]

**Comment by ljmoody** [comment]
I'd like to watch the demo of MTL. Is it possible to watch by _not_ creating a user account, etc. via this link?: [https://forio.com/app/va/va-psd-demo/mtl_demo.html](url)

**Comment by jamesmrollins** [comment]
Hi @moodylj - This is a link to a working version of the simulation, so nothing to watch.  It is something to interact with.  Certainly, you would have to register.  The registration is good for 90 days.  The videos on the website do not require you to register. 

Does this explanation help?

**Comment by jamesmrollins** [comment]
BTW - If you want an email signal to come to my attention, use the "@" and my username "jamesmrollins" and it will send out a notice via email to me.

**Comment by ljmoody** [comment]
> Hi @moodylj - This is a link to a working version of the simulation, so nothing to watch. It is something to interact with. Certainly, you would have to register. The registration is good for 90 days. The videos on the website do not require you to register.
> 
> Does this explanation help?

Yes, your explanation helps @jamesmrollins and cleared up my misunderstanding. I was going to assign that issue to you, but you found it.
